src/tqec/simulation/io_utils/csv_files.py

```diff
@@ -0,0 +1,55 @@
+"""Read and write simulation data from and to csv files."""
+
+import io
+from collections.abc import Iterable
+from pathlib import Path
+from typing import Literal, TextIO
+
+import sinter
+
+
+def write_sinter_stats_to_csv(
+    filepath: str | Path,
+    stats: Iterable[sinter.TaskStats],
+    if_file_exists: Literal["merge"]
+    | Literal["overwrite"]
+    | Literal["raise"] = "raise",
```

Comment:
Equivalent to (but longer than):

```suggestion
    if_file_exists: Literal["merge", "overwrite", "raise"] = "raise",
```